packages/core/src/projection/ProjectionSystem.ts

```diff
@@ -98,21 +101,21 @@ export class ProjectionSystem extends System
     /**
      * Updates the projection matrix based on a projection frame (which is a rectangle)
      *
-     * @param {PIXI.Rectangle} destinationFrame - The destination frame.
+     * @param {PIXI.Rectangle}[destinationFrame] - The destination frame.
      * @param {PIXI.Rectangle} sourceFrame - The source frame.
      * @param {Number} resolution - Resolution
      * @param {boolean} root - If is root
      */
-    calculateProjection(destinationFrame: Rectangle, sourceFrame: Rectangle, resolution: number, root: boolean): void
+    calculateProjection(_: Rectangle, sourceFrame: Rectangle, __: number, root: boolean): void
```

Comment:
It's good to keep the names, but we can just prepend with an underscore to ignore from linting for unused params.
```suggestion
    calculateProjection(_destinationFrame: Rectangle, sourceFrame: Rectangle, _resolution: number, root: boolean): void
```